charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -258,6 +258,10 @@ spec:
           {{- with .Values.env.liveness_probe }}
           env: {{ toYaml . | nindent 12 }}
           {{- end }}
+
+        {{- if .Values.controller.additionalContainers }}
```

Comment:
Could you use `with` instead of `if` for readability, like other fields?

Reply:
done